src/test/coins_tests.cpp

```diff
@@ -64,7 +64,7 @@ class CCoinsViewTest : public CCoinsView
                     map_.erase(it->first);
                 }
             }
-            mapCoins.erase(it++);
+            it = mapCoins.erase(it);
```

Comment:
Be aware that there is a bug in robin_hood map https://github.com/martinus/robin-hood-hashing/issues/42 (as well as in tessil's map https://github.com/Tessil/robin-map/issues/20) where it is possible that entries are iterated over a second time before `end()` is reached.